app/gui/view/documentation/src/lib.rs

```diff
@@ -315,29 +312,26 @@ impl View {
             );
 
 
-            // === Hovered item preview caption ===
-
-            spring_muliplier <- style.update.map(|s| s.caption_animation_spring_multiplier);
-            caption_anim.set_spring <+ spring_muliplier.map(|m| Spring::default() * m);
-            show_caption <- frp.show_hovered_item_preview_caption.on_true();
-            hide_caption <- frp.show_hovered_item_preview_caption.on_false();
-            caption_anim.target <+ show_caption.constant(1.0);
-            caption_anim.target <+ hide_caption.constant(0.0);
-            _eval <- all_with(&caption_anim.value, &style.update, f!((value, style) {
-                model.set_caption_height(value * style.caption_height, style)
-            }));
-
-
             // === Size ===
 
             size <- style.update.map(|s| Vector2(s.width, s.height));
             eval size((size) model.set_size(*size));
 
+
             // === Style ===
 
             eval style.update((style) model.update_style(*style));
 
 
+            // === Show/hide animation ===
+
+            width_anim.target <+ frp.show.constant(1.0);
+            width_anim.target <+ frp.hide.constant(0.0);
+            width_anim.skip <+ frp.skip_animation;
+            width_anim.target <+ init.constant(1.0);
```

Comment:
Instead of using this `init.constant`, the state could be initialized by emitting `show()` (or `set_visible(true)`) at the bottom of this `init` function.

Reply:
Even better, we can create `Animation` with `new_with_init` constructor.